AsmSpy/Program.cs

```diff
@@ -87,22 +87,24 @@ public static void AnalyseAssemblies(DirectoryInfo directoryInfo, bool onlyConfl
             if (onlyConflicts)
                 Console.WriteLine("Detailing only conflicting assembly references.");
 
-            foreach (var assembly in assemblies)
+            foreach (var assemblyName in assemblies.Keys.OrderBy(asm => asm, StringComparer.OrdinalIgnoreCase))

```

Comment:
Why not just write `var assembly in assemblies.OrderBy(x => x.Key, StringComparer.OrdinalIgnoreCase)`
Then you wouldn't have to change the following lines.